SEO excerpts: Use part 4 (#3838)
timescale/docs: use-timescale/extensions/index.md

---
title: PostgreSQL extensions

> excerpt: Timescale Cloud comes with a number of Timescale and PostgreSQL extensions enabled by default. See what other extensions you can enable for your service

Reviewer: ```suggestion
excerpt: Timescale Cloud comes with a number of Timescale and PostgreSQL extensions enabled by default. See all the extensions you can enable for your service
```
